tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -563,7 +563,7 @@ public function testErrorsWithLastRule()
         $errors = $validator->errors(['email' => 'not an email!']);
         $expected = [
             'email' => [
-                'alpha' => 'The provided value is invalid'
+                'alpha' => 'alpha'
```

Comment:
Probably, but showing `The provided value is invalid` isn't particularly useful to the end user either as gives no info about actually what validation failed.